perun-wui-registrar/src/main/java/cz/metacentrum/perun/wui/registrar/widgets/items/Selectionbox.java

```diff
@@ -127,8 +134,15 @@ public Widget getWidget() {
 	@Override
 	protected void setValueImpl(String value) {
 		if (isOnlyPreview()) {
-			getPreview().add(new Span(value));
-			return;
+			Map<String, String> opts = parseItemOptions();
```

Comment:
This fails if previously selected value/option is removed from the form item definition (form modified by VO manager). I suggest in such case to fallback to add such value between the options locally (even if we have only "raw" value and not display value).

Reply:
Select items should work correctly now also if the selected option was deleted by the manager.

I also changed the color of the caret-down to light gray, so it can be more clear, that these items are not editable on the preview.